src/frontend/utils/dato/dato.ts

```diff
@@ -93,3 +93,18 @@ export const erIsoStringGyldig = (isoString?: IsoDatoString): boolean => {
     const dato = parseISO(isoString);
     return isValid(dato);
 };
+
+export const parseTilOgMedDato = (tom: IsoDatoString | undefined) =>
+    isoStringTilDateMedFallback({
+        isoString: tom,
+        fallbackDate: tidenesEnde,
+    });
+
+export const parseFraOgMedDato = (fom: IsoDatoString | undefined) =>
+    isoStringTilDateMedFallback({
+        isoString: fom,
+        fallbackDate: tidenesMorgen,
+    });
```

Comment:
Lurer på om jeg foretrekker om denne er i den originale filen hvis denne kun blir brukt i én fil 🤔

Reply:
Jeg liker å samle utils, men det er helt greit, kan flytte dem tilbake 😄 da jeg flyttet dem hit trodde jeg at de skulle brukes flere steder, men endte med at det bare ble ett sted

Reply:
Hvis jeg flytter `erSammeFom` til EkspanderbartBegrunnelsePanel blir disse utilsene brukt i to filer